packages/mui-base/src/Accordion/Root/useAccordionRoot.ts

```diff
@@ -0,0 +1,261 @@
+'use client';
+import * as React from 'react';
+import { mergeReactProps } from '../../utils/mergeReactProps';
+import { NOOP } from '../../utils/noop';
+import { useControlled } from '../../utils/useControlled';
+import { ARROW_DOWN, ARROW_UP, ARROW_RIGHT, ARROW_LEFT } from '../../Composite/composite';
+
+const SUPPORTED_KEYS = [ARROW_DOWN, ARROW_UP, ARROW_RIGHT, ARROW_LEFT, 'Home', 'End'];
+
+function getActiveTriggers(accordionSectionRefs: {
+  current: (HTMLElement | null)[];
+}): HTMLButtonElement[] {
+  const { current: accordionSectionElements } = accordionSectionRefs;
+
+  const output: HTMLButtonElement[] = [];
+
+  for (let i = 0; i < accordionSectionElements.length; i += 1) {
+    const section = accordionSectionElements[i];
+    if (!isDisabled(section)) {
+      const trigger = section?.querySelector('[type="button"]') as HTMLButtonElement;
+      if (!isDisabled(trigger)) {
+        output.push(trigger);
+      }
+    }
+  }
+
+  return output;
+}
+
+function isDisabled(element: HTMLElement | null) {
+  return (
+    element === null ||
+    element.hasAttribute('disabled') ||
+    element.getAttribute('data-disabled') === 'true'
+  );
+}
+/**
+ *
+ * Demos:
+ *
+ * - [Accordion](https://mui.com/base-ui/react-accordion/#hook)
+ *
+ * API:
+ *
+ * - [useAccordionRoot API](https://mui.com/base-ui/react-accordion/hooks-api/#use-accordion-root)
+ */
+export function useAccordionRoot(
+  parameters: useAccordionRoot.Parameters,
```

Comment:
```suggestion

export function useAccordionRoot(
  parameters: useAccordionRoot.Parameters,
```

We no longer document hooks on the website